experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -414,6 +414,20 @@ describe('Node SDK', () => {
       assert.equal(actualContextManager, expectedContextManager);
       await sdk.shutdown();
     });
+
+    it('should register a propagators as defined in OTEL_PROPAGATORS if trace SDK is configured', async () => {
+      process.env.OTEL_PROPAGATORS = 'b3';
+      const sdk = new NodeSDK({
+        traceExporter: new ConsoleSpanExporter(),
+        autoDetectResources: false,
+      });
+
+      sdk.start();
+
+      assert.deepStrictEqual(propagation.fields(), ['b3']);
+
+      await sdk.shutdown();
```

Comment:
nit:  Perhaps clean up the envvar for subsequent tests, in case it matters.

```suggestion
      await sdk.shutdown();
      delete process.env.OTEL_PROPAGATORS;
```

Reply:
Ah yes - good catch!